bindings/pyroot/pythonizations/doc/index.md

```diff
@@ -0,0 +1,4 @@
+\defgroup Pythonizations PyROOT pythonizations
```

Comment:
```suggestion
\defgroup Pythonizations Python interface
```

I believe we want to give a very generic title, then explain what "pythonizations" are in the comment below.